sky/core.py

```diff
@@ -524,9 +524,13 @@ def queue(cluster_name: str,
 
 @usage_lib.entrypoint
 # pylint: disable=redefined-builtin
-def cancel(cluster_name: str,
-           all: bool = False,
-           job_ids: Optional[List[int]] = None) -> None:
+def cancel(
+    cluster_name: str,
+    all: bool = False,
+    job_ids: Optional[List[int]] = None,
+    # pylint: disable=invalid-name
+    _ignore_cluster_aliveness: bool = False,
```

Comment:
Can we add this to Args:?

Reply:
Done. Thanks!

Reply:
How about "_try_cancel_if_cluster_is_init()"?